### ISSUE TITLE
Cloud Training EfficientDet (d4)

### ISSUE THREAD

**Issue by aleksandar** [tensorflow/models#8949]
I've tried to train a efficientdet-d4 model, based on [a config](https://github.com/tensorflow/models/blob/master/research/object_detection/configs/tf2/ssd_efficientdet_d4_1024x1024_coco17_tpu-32.config). Because configs for efficientdets and tf2 are just for TPUs I tried this [option](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_training_and_evaluation.md#training-with-tpu), right starting ckpt (D4) was used from [the model zoo](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_detection_zoo.md). 

`# From the tensorflow/models/research/ directory
cp object_detection/packages/tf2/setup.py .
gcloud ai-platform jobs submit training `whoami`_object_detection_`date +%m_%d_%Y_%H_%M_%S` \
    --job-dir=gs://${MODEL_DIR} \
    --package-path ./object_detection \
    --module-name object_detection.model_main_tf2 \
    --runtime-version 2.1 \
    --python-version 3.6 \
    --scale-tier BASIC_TPU \
    --region us-central1 \
    -- \
    --use_tpu true \
    --model_dir=gs://${MODEL_DIR} \
    --pipeline_config_path=gs://${PIPELINE_CONFIG_PATH}`

This example from [the docs .md ](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_training_and_evaluation.md#training-with-tpu) throws several error before the last one:

- python version should be 3.7
- problem with accessing a restricted bucket (it seams as an open issue regarding tpu cloud train...)

Current obstacle is:
![Screenshot from 2020-07-23 15-34-34](https://user-images.githubusercontent.com/229280/88292529-08380200-ccfa-11ea-9390-247c0f242c81.png)

Does anybody experienced something similar?

Thanks,
Alex

**Comment by syiming** [comment]
Hi, could you please try to remove the restrictions and retry it? And could you please provide more details about the error message? Thanks.

**Comment by aleksandar** [comment]
the restrictions? This is the only error messages I got.

**Comment by syiming** [comment]
You are saying you encounter an error with accessing a restricted bucket. Can you try to remove that restriction? Also, you can click the log to view more details. Thanks.

**Comment by aleksandar** [comment]
Yes, I made the bucket open, this is not an issue now. Besides this, this screenshot already presents log detail for the job. When hit  the log link from this top row, I get same log page, with same log query parameters in the header.

**Comment by aleksandar** [comment]
Here is place with error type messages starting to arrive
![Screenshot from 2020-07-25 17-13-13](https://user-images.githubusercontent.com/229280/88460148-5e43ab80-ce9a-11ea-962c-18b439354dcb.png)

Last errors before ones with tensor values
![Screenshot from 2020-07-25 17-13-40](https://user-images.githubusercontent.com/229280/88460159-79162000-ce9a-11ea-83cf-1c217387c165.png)

On the first image the first error is 'ERROR: tensorflow-cpu 2.1.1 has requirement gast==0.2.2, but you'll have gast 0.3.3 which is incompatible.' It is a little bit strange that TF version is in the root of the issue.
